docs/docs-content/clusters/cluster-management/certificate-management.md

```diff
@@ -31,9 +31,11 @@ You can learn more about each option in the following sections.
 
 ## Limitations
 
-- Neither automatic nor manual certificate renewal through the Palette UI is supported by Edge clusters. For Edge airgap
-  clusters, you can renew the certificates from Local UI. Refer to
-  [Renew Certificates for Airgapped Edge Hosts](../edge/cluster-management/certificate-renewal.md) for more information.
+- Both automatic and manual certificate renewal will renew control plane certificates only. Worker node certificate
+  renewal is not supported.
+
+- Airgapped Edge clusters are not managed by Palette. To renew the certificates for airgapped Edge clusters, refer to
```

Comment:
I'm curious about this one. Who manages these clusters, then? :D Or is Palette Saas the one that doesn't manage the clusters?

Reply:
I think it would be Local UI

Reply:
Yes it is managed locally. They don't have to use local UI - they may use the API or just use the cloud-init stages and leave it alone once it's up. But since Palette doesn't have a connection to airgapped Edge hosts, it cannot manage these instances.